starknet_py/net/client.py

```diff
@@ -203,7 +203,7 @@ async def estimate_fee(
     @abstractmethod
     async def call_contract(
         self,
-        invoke_tx: Union[InvokeFunction, Call],
+        invoke_tx: Call,
```

Comment:
Maybe it is the best moment to change the name of this parameter? 🙏  `invoke_tx` is completely misleading 😓

Reply:
Renamed, though I'm unsure about this being even more breaking (no easy way of deprecating parameters in python though).